src/app/modules/image-viewer/image-viewer-popover/image-viewer-popover.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component } from '@angular/core';
 
 import { ImageViewerData } from 'ccf-database';
+import { ImageViewerLayer } from 'src/app/core/models/image-viewer-layer';
```

Comment:
Fix import

Reply:
Weird that the auto import did that.  Good catch.